### DISCUSSION TITLE
IP Config & Label on Browser

### DISCUSSION THREAD

**Discussion by diman64** [arendst/Tasmota#13855] [Q&A] [answered]
Hi to all,
I have an question as per object, if is possible to implemented the manual IP in the Wi-Fi configuration instead of DHCP in the next FW, to have two configuration options. This is because, once you have configured the Wi-Fi from the information you have the current IP from the DHCP, but if you disconnect the IP may have changed and it becomes a problem to find the new IP with the browser.
Therefore, it would be better to implement the option to disable DHCP and manually enter the IP address, subnet mask etc ...
Second question, if is possible to rename the label on browser like a picture, because the label name is standard for all toggle1 toggle2 etc.. it would be nice to be able to describe each option with its real name even through the browser.
That's all.
regards
![Cattura](https://user-images.githubusercontent.com/90111963/143864265-dfdf3fb7-0567-424e-95fe-f097473f2cfc.PNG)

**Comment by sfromis** [comment, marked as answer]
Using the command [WebButton](https://tasmota.github.io/docs/Commands/#webbutton), you can change button labels. Nothing stops you from using HTML there, if you want images. Note that the web UI is *not* meant for daily operation, more for admin/debug only.

The IP address from a DHCP server should not change right away on a just-registered device, as it should have a lease time in which the IP is reserved.

**Reply by Jason2866** [reply]
You can configure the IP addresses manually. See the documentation. There will be no change in WebUi for doing manual IP config.

**Reply by diman64** [reply]
Hi bro, yes I know I can reserve the IP from router config but I prefer to select it from the tasmota fw, aniway it's ok.
thks
